rest-servlet/src/main/java/org/hawkular/inventory/rest/RestTenants.java

```diff
@@ -100,4 +134,20 @@ public Response delete() {
         return Response.noContent().build();
     }
 
+    @DELETE
+    @Path("/")
+    @ApiOperation("Deletes the tenant and all its data. Be careful!")
+    @ApiResponses({
+                          @ApiResponse(code = 204, message = "OK"),
+                          @ApiResponse(code = 401, message = "Unauthorized access"),
+                          @ApiResponse(code = 404, message = "Tenant doesn't exist", response = ApiError.class),
+                          @ApiResponse(code = 500, message = "Server error", response = ApiError.class)
+                  })
+    public Response delete(@PathParam("tenantId") String tenantId) {
```

Comment:
`tenantId` is not mentioned in the `@Path` annotation on the method. Is it correct to mention it here? Also for consistency with `GET /tenants/{tenantId}`, should this be `@Path("/tenants/{tenantId}")`?